connectors/concur/src/main/java/com/vmware/connectors/concur/ConcurController.java

```diff
@@ -7,7 +7,7 @@
 
 package com.vmware.connectors.concur;
```

Comment:
Could you fix the copyright file header above while you're re-org'ing.  Maybe do a search through the code in case we missed some other places too.

Reply:
I'll get this in the followup ticket/PR.